CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-07-16
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240716.
+
+### Announcements
+
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta APIs will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta APIs closer to the 1.30 release.
+* 1.30 is the next LTS version after 1.27. Upgrade from 1.27 LTS to 1.30 LTS will be possible starting August 2024. More information about AKS LTS is available [here](https://learn.microsoft.com/azure/aks/long-term-support).
+* 1.27 community supported has ended. The ability to exit [Long-Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support) on 1.27 is blocked. To exit, upgrade to 1.28 and disable LTS with the `az aks upgrade` command.
```

Comment:
'The ability to exit [Long-Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support) on 1.27 is blocked' sounds a bit ominous whereas the line right after on upgrading to 1.28 and disabling sounds more positive. Can we reframe this entire point?